Docs/MoltenVK_Runtime_UserGuide.md

```diff
@@ -368,6 +368,7 @@ In addition to core *Vulkan* functionality, **MoltenVK**  also supports the foll
 - `VK_INTEL_shader_integer_functions2`
 - `VK_NV_fragment_shader_barycentric`
   - *Requires Metal 2.2 on Mac or Metal 2.3 on iOS.*
+- `VK_EXT_external_memory_metal`
```

Comment:
Like in `MVKExtensions.def`, extensions here are sorted in a particular order:
* Khronos-ratified (`KHR`)
* Multivendor (`EXT`)
* Vendor-specific, alphabetically.

Within each group, extensions are sorted alphabetically.

Reply:
I'll fix this in another PR.